ConwaysGameOfLife/Universe.cs

```diff
@@ -65,7 +71,7 @@ public List<Location> GetCellNeighbourLocations(Cell cell)
         public int CountLiveNeighbours(Cell cell)
         {
             var aliveNeighbours = new List<Location>();

```

Comment:
alternative: `var aliveNeighbours = 0;`
then do the check in the loop and increment by 1 if the cell is alive

Reply:
can also rejig this function (with some more widespread refactoring) to do a Linq count. we can discuss this tomorrow